sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CapacityReservationGroupInstanceViewTypes.cs

```diff
@@ -0,0 +1,54 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Compute.Models
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System.Runtime;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Defines values for CapacityReservationGroupInstanceViewTypes.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum CapacityReservationGroupInstanceViewTypes
```

Comment:
We do not want pure Enums in .Net SDK. Adding new values to them in the future will be breaking changes. 
Go back to the Swagger file, set the value "modelAsString" for the enum "[CapacityReservationGroupInstanceViewTypes](https://github.com/Azure/azure-rest-api-specs/pull/15100/files#diff-b0c8b989f3e68f5a826a676868ed64d355d0441bcf390f440e9d7dd85ba2c5bbR4265)" to 'true' instead of 'false'.
Then regenerate the .Net SDK.